scripts/entrypoint.sh

```diff
@@ -70,8 +71,8 @@ else
 fi
 
 export UPDATE_BRANCH
-echo "Branch to update is ${UPDATE_BRANCH}"
-git checkout -b ${UPDATE_BRANCH}
+printf 'Branch to update is %s\n' "${UPDATE_BRANCH}"
+git checkout -b "${UPDATE_BRANCH}"
 git branch
```

Comment:
This is something that I did but I'm just seeing now - is there ever a situation when the user would want to be using an update branch that already exists? E.g., so they would do just a checkout without -b, and then possibly a pull then push?

Reply:
I’m not sure; I, too, was a bit surprised by the branch usage. I would
have expected this to just be committed onto a detached head and pushed
with `HEAD:refs/heads/update-"$(date -u +%s)"`.

Reply:
I'd be good if you wanted to change this behavior. Right now we assume the branch doesn't exist - for the most part this will be the case, but as an example, @Beanow wants to use a cron that runs every 6 hours. He'd either need a more detailed timestamp, an automated merge and delete of the branch (instead of pull request) or a fix as you suggested.